app/views/collections/_meta.html.erb

```diff
@@ -1,17 +1,18 @@
 <% title title_string %>
-<link rel="canonical" href="<%= app_url("/about") %>" />
-<meta name="description" content="<%= title_string %> on <%= community_name %>">
+
+<link rel="canonical" href="<%= app_url(path) %>" />
```

Comment:
@rhymes, quick question. Does a canonical URL make sense in the context of a series? As far as I know we don't have a mechanism to specify a canonical URL for a series, only a post.

Reply:
Good point, you're right, I don't think a Forem series can have a duplicate place on the web, it's not like a post that can come from multiple places.

@jeferson-sb would it be okay for you to remove the canonical URL from the tags?